command.md

```diff
@@ -2943,6 +2943,13 @@ values each time it is called.</p>
 <ul>
 <li><a name="get_directories.0"></a> list&lt;(<a href="#descriptor"><a href="#descriptor"><code>descriptor</code></a></a>, <code>string</code>)&gt;</li>
 </ul>
+<h4><a name="initial_cwd"><code>initial-cwd: func</code></a></h4>
+<p>Return a preopen for use as the program's initial current working
```

Comment:
I wonder why we are returning a pre-open and what I believe to be the path, instead of just the path. It seems the guest will need to resolve pre-opens for path resolution anyway. Adding a pre-open FD here seems to give a chance or at least a question of why resolution of the pwd would happen in a different way that a normal path lookup in the guest. It also forces the host to search the host-side pre-opens for that dir which feels unnecessary.

TL;DR' what's motivating the descriptor part of the tuple and what would happen if it wasn't here?

Reply:
No strong reason; the cwd in POSIX acts a lot like a file descriptor that the OS holds for you, so it seemed natural to say there should be a file descriptor here too. But you're right, if someone wants that, they can pass in a regular preopen for it. I've now removed the descriptor here.